test/fixtures/postgresql-ha/main.tf

```diff
@@ -114,6 +114,10 @@ module "pg" {
   user_password = "foobar"
 
   additional_users = [
+    {
+      name     = "tftest1"
+      password = "raboof"
+    },
```

Comment:
What's the motivation to add this ?

Reply:
To fix the test logic. The test to verify the user count looks for four users with that start with `tftest`: https://github.com/GoogleCloudPlatform/terraform-google-sql-db/blob/627ae502386e263c1f48073a2b7ba9db408ee5fd/test/integration/postgresql-ha/controls/pg.rb#L99-L103

As it stands, the test only added three users and the user names are tftest, tftest2, and tftest3. I figured the omission of tftest1 was an oversight.

Reply:
Nope it has already been fixed by https://github.com/GoogleCloudPlatform/terraform-google-sql-db/commit/aec7fdb6f3c0de16c64141b4979537f916573811

You should have rebased the master branch.
I'll revert this.